lib/slack.py

```diff
@@ -84,6 +90,7 @@ def __build_message__(self, suggested_video, channel, username, search_id, index
 
         self.buttons[0]['value'] = id
         self.buttons[1]['value'] = '{{"index": {}, "search_id": {}}}'.format(index, search_id)
```

Comment:
No need to have the index stored in this dict since you're storing it in Next button's value